build_all.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -eux
+
+for pb in $(find packagingbuild -name Dockerfile); do
+	path=$(dirname $pb)
```

Comment:
Noticed tabs are used here for indentation.

Would be nice to follow the same format with 2 spaces as a separator for `bash` files.
That'll make all the scripts consistent across the fleet.